### PR TITLE
Updated shellmap.

### PR DESCRIPTION
**Pull request by Dzierzan**

### CODE REVIEW

File: mods/hv/maps/shellmap/map.yaml [hunk 1]
```diff
@@ -26,12 +26,12 @@ Players:
 		Name: Creeps
 		NonCombatant: True
 		Faction: Random
-	PlayerReference@Yuruki:
+	PlayerReference@Yuruki Industries:
```
Comment (Mailaender) [inline mods/hv/maps/shellmap/map.yaml:29]: I guess this is an automated change.

**Comment by Mailaender** [comment]
image

File: mods/hv/maps/shellmap/map.yaml (hunk 1 above)
Comment (Dzierzan) [reply mods/hv/maps/shellmap/map.yaml:29]: Yes

**Comment by Dzierzan** [comment]
I forgot about these two.

**Pull request merged**